pkg/agent/flowexporter/connections/conntrack_linux.go

```diff
@@ -0,0 +1,156 @@
+// +build linux
+
+package connections
+
+import (
+	"net"
+
+	"github.com/ti-mo/conntrack"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/config"
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+)
+
+var _ ConnTrackPoller = new(connTrackPoller)
+
+type connTrackPoller struct {
+	nodeConfig  *config.NodeConfig
+	serviceCIDR *net.IPNet
+	conntrack   ConnTrack
+}
+
+func NewConnTrackPoller(nodeConfig *config.NodeConfig, serviceCIDR *net.IPNet, conntrack ConnTrack) *connTrackPoller {
```

Comment:
@srikartati I feel like the `ConnTrackPoller` name is not very appropriate here since this object is not actually doing any polling

Reply:
I agree with your observation. Swapped the names of connTrackPoller and connTrack to make them a better match.